modules/rostests/apitests/localspl/tests.c

```diff
@@ -210,6 +210,14 @@ _RunRemoteTest(const char* szTestName)
 
 START_TEST(fpEnumPrinters)
 {
+#if defined(_M_AMD64)
+    if (!winetest_interactive)
+    {
+        skip("ROSTESTS-366: Skipping localspl_apitest:fpEnumPrinters because it hangs on Windows Server 2003 x64-Testbot. Set winetest_interactive to run it anyway.\n");
```

Comment:
All localspl API tests hang on the Win2003 x64 Testbot the same way. It's just that you won't notice now as we're running tests under a restricted user account.

Reply:
Ok, will skip all of them.